app/scraper/loot/steam_loot.py

```diff
@@ -13,6 +12,7 @@
 from app.scraper.info.steam import skip_age_verification
 from app.scraper.info.utils import clean_loot_title
 from app.scraper.loot.scraper import Scraper
+from app.scraper.loot.steam_games import SteamRawOffer

```

Comment:
The scrapers are meant to be independent of each other, please don't cross reference them